config/initializers/hesa_ethnicities.rb

```diff
@@ -72,11 +72,12 @@ module HesaEthnicityCollections
     ['999', HesaEthnicityValues::NOT_AVAILABLE],
   ].freeze
 
-  HESA_ETHNICITIES_2023_2024 = HESA_ETHNICITIES_2022_2023
-
   # https://www.hesa.ac.uk/collection/c20053/e/ethnic
   HESA_ETHNICITIES_2020_2021 = HESA_ETHNICITIES_2019_2020
   # Unchanged from 2020-2021
   # https://www.hesa.ac.uk/collection/c21053/e/ethnic
   HESA_ETHNICITIES_2021_2022 = HESA_ETHNICITIES_2020_2021
+
+  HESA_ETHNICITIES_2023_2024 = HESA_ETHNICITIES_2022_2023
+  HESA_ETHNICITIES_2024_2025 = HESA_ETHNICITIES_2022_2023
```

Comment:
No, Hesa sex has a fallback to look at the latest available constant. I consider at first to add a fallback here, but then I disregard because it would be challenging and I didn't want to scope creep this PR.